### PR TITLE
OSDOCS#40868 - Updating runbook links & abstract

### PR DESCRIPTION
**Pull request by jherrman**
Version(s):
4.17

Issue:
https://issues.redhat.com/browse/CNV-40868

Link to docs preview:
TBA

QE review:
- [ ] QE has approved this change.

### CODE REVIEW

File: virt/monitoring/virt-runbooks.adoc [hunk 1]
```diff
@@ -7,9 +7,9 @@ include::_attributes/common-attributes.adoc[]
 toc::[]
 
 :!virt-runbooks:
-Runbooks for the {VirtProductName} Operator are maintained in the link:https://github.com/openshift/runbooks/tree/master/alerts/openshift-virtualization-operator[openshift/runbooks] Git repository, and you can view them on GitHub. To diagnose and resolve issues that trigger {VirtProductName} xref:../../observability/monitoring/managing-alerts.adoc#managing-alerts[alerts], follow the procedures in the runbooks.
+To diagnose and resolve issues that trigger {VirtProductName} xref:../../observability/monitoring/managing-alerts.adoc#managing-alerts[alerts], follow the procedures in the runbooks for the {VirtProductName} Operator. {VirtProductName} alerts are displayed in the *Virtualization* -> *Overview* tab in the web console.
```
Comment (sradco) [inline virt/monitoring/virt-runbooks.adoc:10]: @jherrman 
All firing alerts are at `observe ->  alerts`
Even-though they can also be found in  `Virtualization* -> *Overview` , I think we should replace this to the general OCP alerts page.

Comment (jherrman) [reply virt/monitoring/virt-runbooks.adoc]: Thank you, updated now.

File: virt/monitoring/virt-runbooks.adoc [hunk 2]
```diff
@@ -219,6 +219,11 @@ Runbooks for the {VirtProductName} Operator are maintained in the link:https://g
 
 * link:https://github.com/openshift/runbooks/blob/master/alerts/openshift-virtualization-operator/SSPTemplateValidatorDown.md[View the runbook] for the `SSPTemplateValidatorDown` alert.
 
+[id="virt-runbook-SSPOperatorDown"]
```
Comment (ShaunaDiaz) [inline virt/monitoring/virt-runbooks.adoc:222]:
```suggestion
[id="virt-runbook-ssp-operatordown_{context}"]
```
Needs context tag, and likely, lowercase.
See https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-titles-and-section-headings. Although not listed there, if you scroll up, all of the guidance around IDs for assemblies and modules stipulate lowercase in IDs.

**Comment by ShaunaDiaz** [comment]
@jherrman Please squash the commits and get QE ack prior to merge. TY!

File: virt/monitoring/virt-runbooks.adoc (hunk 2 above)
Comment (jherrman) [reply virt/monitoring/virt-runbooks.adoc:222]: @ShaunaDiaz I followed the ID pattern established in the module - https://raw.githubusercontent.com/openshift/openshift-docs/main/virt/monitoring/virt-runbooks.adoc

Still, you have a point that it doesn't adhere to the naming conventions, I'll fix it in another commit.

**Pull request merged**